WordPress/WordPressTest/EditorSettingsTests.swift

```diff
@@ -0,0 +1,65 @@
+import XCTest
+@testable import WordPress
+
+class EditorSettingsTests: XCTestCase {
+    override func setUp() {
+        super.setUp()
+
+        let userDefaults = UserDefaults.standard
+        userDefaults.removeObject(forKey: "kUserDefaultsNativeEditorAvailable")
```

Comment:
Why clean the standard UserDefaults up if you are using the EphemeralKeyValyeDatabase on the tests?